zou/app/blueprints/source/csv/assets.py

```diff
@@ -68,6 +141,10 @@ def import_row(self, row, project_id):
                     "id"
                 ]
             episode_id = self.episodes.get(episode_name, None)
+        elif episode_name is not None:
+            raise ImportRowException(
```

Comment:
Same here. You can't raise exception in the midlle of the code without handling them somewhere.

Reply:
I use RowException now that is handled in the main class